packages/store/addon/-private/system/model/internal-model.ts

```diff
@@ -36,6 +32,19 @@ import { StableRecordIdentifier } from '../../ts-interfaces/identifier';
 import { internalModelFactoryFor, setRecordIdentifier } from '../store/internal-model-factory';
 import CoreStore from '../core-store';
 import coerceId from '../coerce-id';
+import recordDataFor from '../record-data-for';
+
+type DefaultRecordData = import('@ember-data/record-data/-private').RecordData;
+
+function relationshipsFor(instance) {
+  let recordData = recordDataFor(instance) || instance;
+
+  return recordData._relationships;
+}
+
+function relationshipStateFor(instance, propertyName) {
```

Comment:
Besides my comments above, presumably `propertyName` should be `: string` here?

Reply:
yes